08_voting/index.tex

```diff
@@ -1 +1,101 @@
 \chapter{Voting}
+\section{Voting Scenarios}
+\label{sec:VotingScenarios}
+
+Within the context of the coursework, robust voting systems are designed for deciding the outcome of collective actions (e.g. elections for legislative, executive and judicial branches in IIGO).
+When voting is called in IIGO, the President, Speaker, Judge and the islands in the archipelago (i.e. the \emph{Voters}, assuming that no diplomatic sanction is in place) will be empowered to proceed with the following series of actions:
```

Comment:
Given IIGO stands for Inter Island Governmental Organisation should we say the IIXO or IIGO (not major but would be nice to be consistent across the doc) - other doc editors chime in here pls

Reply:
what? what is IIXO??

Reply:
btw it is 
"**Inter-Island** Governmental Organisation"

Reply:
it is ''the IIGO''

Reply:
changed - "the IIGO"